plugin/documents.py

```diff
@@ -491,6 +491,9 @@ def on_text_command(self, command_name: str, args: Optional[dict]) -> Optional[T
         return None
 
     def on_post_text_command(self, command_name: str, args: Optional[Dict[str, Any]]) -> None:
+        if command_name == 'paste' and userprefs().format_on_paste:
```

Comment:
There exists also a command `paste_and_indent` (<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>V</kbd>).
Should it run the formatting too, when that command was invoked?

By the way, the next `if` can be `elif` now ;-)

Reply:
>  Should it run the formatting too, when that command was invoked?

I would not add it, but don't have any strong opinions about it.
If you think that formatting should happen, I will add it.